Travis failure:
```
../../src/net_processing.cpp:2999:54: error: request for member ‘round’ in ‘filterRounder’, which is of non-class type ‘FeeFilterRounder(CFeeRate)’
                 CAmount filterToSend = filterRounder.round(currentFilter);
```

--- a/src/net_processing.cpp
+++ b/src/net_processing.cpp
@@ -2995,7 +2995,8 @@ bool SendMessages(CNode* pto, CConnman& connman)
             CAmount currentFilter = mempool.GetMinFee(GetArg("-maxmempool", DEFAULT_MAX_MEMPOOL_SIZE) * 1000000).GetFeePerK();
             int64_t timeNow = GetTimeMicros();
             if (timeNow > pto->nextSendTimeFeeFilter) {
-                static FeeFilterRounder filterRounder(::minRelayTxFee);
+                static CFeeRate default_feerate(DEFAULT_MIN_RELAY_TX_FEE);
+                static FeeFilterRounder filterRounder(default_feerate);
                 CAmount filterToSend = filterRounder.round(currentFilter);
                 if (filterToSend != pto->lastSentFeeFilter) {
                     connman.PushMessage(pto, msgMaker.Make(NetMsgType::FEEFILTER, filterToSend));

